ingest/Snakefile

```diff
@@ -0,0 +1,18 @@
+# USAGE: snakemake --snakefile Snakefile -c 2 pull_data
+
+from doctest import debug_script
+from snakemake.utils import min_version
+
+min_version("6.0")
+
+module vipr_workflow:
+    snakefile:
+        github("j23414/test_snakemake_6", path="workflow/vipr.smk", branch="main")
```

Comment:
Linking here for easy access: https://github.com/j23414/test_snakemake_6/blob/main/workflow/vipr.smk